### PR TITLE
docs: update Russian translation for ru/learn/common_questions/how_does_the_internet_work/index.md

### PR DESCRIPTION
**Pull request by impocode**

### CODE REVIEW

**Comment by impocode** [comment]
@sashasushko

Привет! Очень жду обратную связь)

**Comment by sashasushko** [comment]
@impocode thank you for your help. Sorry for long response. I remember about your PR. And I'll review it ASAP. Peace 🕊

**Comment by sashasushko** [comment]
It's ok. I'll resolve conflicts.

File: files/ru/learn/common_questions/how_does_the_internet_work/index.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+![Маршрутизатор подключенный к модему](internet-schema-6.png)
+
+Итак, мы подключены к телефонной инфраструктуре. Следующим шагом будет отправка сообщений из нашей сети в сеть, к которой мы хотим подключиться. Для этого мы подключим нашу сеть к интернет-провайдеру (Internet Service Provider (ISP)). Интернет-провайдер — это компания, которая управляет некоторыми специальными _маршрутизаторами_, которые связаны друг с другом и также могут получать доступ к маршрутизаторам других провайдеров. Таким образом, сообщение из нашей сети передается через сеть сетей интернет-провайдеров в сеть назначения. Интернет состоит из всей этой инфраструктуры сетей.
+
+![Полный Интернет стек](internet-schema-7.png)
+
+### Поиск компьютеров
+
+Если вы хотите отправить сообщение на другой компьютер, вы должны указать, какому из них. Поэтому каждый компьютер, подключенный к сети, имеет свой уникальный адрес, который идентифицирует его, называемый "IP-адресом" (где IP означает _Internet Protocol_(Интернет-протокол, протокол интернета или межсетевой протокол)). Это адрес, состоящий из четырех чисел, разделенных точками, например: `192.168.2.10`.
```
Comment (Saionaro) [inline files/ru/learn/common_questions/how_does_the_internet_work/index.md:84]:
```suggestion
Если вы хотите отправить сообщение на другой компьютер, вы должны указать как его найти. Поэтому каждый компьютер, подключенный к сети, имеет свой уникальный адрес, называемый "IP-адресом" (где IP означает _Internet Protocol_ (межсетевой протокол)). Это адрес, состоящий из четырех чисел, разделенных точками, например: `192.168.2.10`.
```

File: files/ru/learn/common_questions/how_does_the_internet_work/index.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+Это совершенно нормально для компьютеров, но нам, людям, трудно запомнить такой адрес. Чтобы упростить задачу, мы можем присвоить IP-адресу удобочитаемое имя, называемое _доменное имя_. Например (на момент написания; IP-адреса могут меняться) `google.com` — это доменное имя, используемое поверх IP-адреса `142.250.190.78`. Таким образом, использование доменного имени является для нас самым простым способом доступа к компьютеру через Интернет.
```
Comment (Saionaro) [inline files/ru/learn/common_questions/how_does_the_internet_work/index.md:86]:
```suggestion
Это совершенно нормально для компьютера, но для человека сложно запомнить такой адрес. Чтобы упростить задачу, мы можем присвоить IP-адресу удобочитаемое имя, называемое _доменное имя_. Например (на момент написания; IP-адреса могут меняться) `google.com` — это доменное имя, используемое для IP-адреса `142.250.190.78`. Таким образом, использование доменного имени является для нас самым простым способом доступа к компьютеру через Интернет.
```

File: files/ru/learn/common_questions/how_does_the_internet_work/index.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+![Вот как доменное имя может быть псевдонимом IP-адреса](dns-ip.png)
+
+### Интернет и веб
```
Comment (Saionaro) [inline files/ru/learn/common_questions/how_does_the_internet_work/index.md:90]:
```suggestion
### Интернет и Веб
```

File: files/ru/learn/common_questions/how_does_the_internet_work/index.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+Как вы могли заметить, когда мы просматриваем Веб с помощью веб-браузера, мы обычно используем доменное имя для доступа к веб-сайту. Означает ли это, что Интернет и Веб — одно и то же? Это не так просто. Как мы видели, Интернет — это техническая инфраструктура, позволяющая соединить вместе миллиарды компьютеров. Среди этих компьютеров некоторые компьютеры (называемые _веб-серверами_) могут отправлять сообщения, понятные веб-браузерам. _Интернет_ — это инфраструктура, тогда как _Веб_ — это служба, построенная поверх инфраструктуры. Стоит отметить, что существует несколько других служб, созданных поверх Интернета, таких как электронная почта и {{Glossary("IRC")}}.
```
Comment (Saionaro) [inline files/ru/learn/common_questions/how_does_the_internet_work/index.md:92]:
```suggestion
Как вы могли заметить, когда мы просматриваем Веб с помощью веб-браузера, мы обычно используем доменное имя для доступа к веб-сайту. Означает ли это, что Интернет и Веб — одно и то же? Не всё так просто. Как мы выяснили ранее, Интернет — это техническая инфраструктура, позволяющая соединять вместе миллиарды компьютеров. Среди этих компьютеров некоторые компьютеры (называемые _веб-серверами_) могут отправлять особые сообщения, понятные веб-браузерам. _Интернет_ — это инфраструктура, тогда как _Веб_ — это служба, построенная поверх инфраструктуры. Стоит отметить, что существует и другие службы, созданные поверх Интернета, такие как электронная почта и {{Glossary("IRC")}}.
```

File: files/ru/learn/common_questions/how_does_the_internet_work/index.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+### Интранет и Экстранет
+
+Интранет — это _частная_ сеть, доступ к которой разрешен только членам определённой организации. Они обычно используются для предоставления участникам портала для безопасного доступа к общим ресурсам, совместной работы и общения. Например, во внутренней сети организации могут размещаться веб-страницы для обмена информацией об отделе или команде, общие диски для управления ключевыми документами и файлами, порталы для выполнения задач делового администрирования и инструменты для совместной работы, такие как вики, доски обсуждений и системы обмена сообщениями.
```
Comment (Saionaro) [inline files/ru/learn/common_questions/how_does_the_internet_work/index.md:96]:
```suggestion
Интранет — это _частная_ сеть, доступ к которой разрешён только членам определённой организации. Такая сеть обычно используется для предоставления участникам портала для безопасного доступа к общим ресурсам, совместной работы и общения. Например, во внутренней сети организации могут размещаться веб-страницы для обмена информацией об отделе или команде, общие диски для управления ключевыми документами и файлами, порталы для выполнения задач делового администрирования и инструменты для совместной работы, такие как базы знаний, доски обсуждений и системы обмена сообщениями.
```

File: files/ru/learn/common_questions/how_does_the_internet_work/index.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+Экстрасети очень похожи на интрасети, за исключением того, что они открывают всю частную сеть или ее часть для обеспечения совместного использования и совместной работы с другими организациями. Обычно они используются для безопасного и надежного обмена информацией с клиентами и заинтересованными сторонами, которые тесно сотрудничают с бизнесом. Часто их функции аналогичны тем, которые предоставляет интранет: обмен информацией и файлами, инструменты для совместной работы, доски обсуждений и т.д.
```
Comment (Saionaro) [inline files/ru/learn/common_questions/how_does_the_internet_work/index.md:98]:
```suggestion

Экстрасети очень похожи на интрасети, за исключением того, что они открывают всю частную сеть или её часть для обеспечения совместного использования и совместной работы с другими организациями. Обычно они используются для безопасного и надёжного обмена информацией с клиентами и заинтересованными сторонами, которые тесно сотрудничают с бизнесом. Часто их функции аналогичны тем, которые предоставляет интранет: обмен информацией и файлами, инструменты для совместной работы, доски обсуждений и т.д.
```

File: files/ru/learn/common_questions/how_does_the_internet_work/index.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+
+И интрасети, и экстрасети работают в той же инфраструктуре, что и Интернет, и используют одни и те же протоколы. Таким образом, авторизованные участники могут получить к ним доступ из разных физических местоположений.
+
+![Графическое представление того, как работают Интранет и Экстранет](internet-schema-8.png)
+
+## Что дальше
```
Comment (Saionaro) [inline files/ru/learn/common_questions/how_does_the_internet_work/index.md:104]:
```suggestion
## Дальнейшие шаги
```

**Comment by Saionaro** [comment]
Due to removed head branch we can't apply some code suggestion changes - I've created a new PR (#12732).
@impocode thank you for your hard work!

**Pull request closed without merging**